### PR TITLE
CBAPI-4902: Expose CBCloud API in read the docs

### PR DESCRIPTION
**Pull request by abowersox-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

https://jira.carbonblack.local/browse/CBAPI-4902

## Pull Request Description

Pull the documentation for `CBCloudAPI` out into its own page, which goes at the front of the "SDK Documentation" section of the docs, as it's key to everything else about the API.  A short introduction and some examples of creating the `CBCloudAPI` object are included.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Full docs have been built locally and verified that they look like they're supposed to.

### CODE REVIEW

File: docs/cbc_sdk.cbcloudapi.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+============================
+
+Authenticate to the Carbon Black Cloud server with directly-supplied parameters:
+
+::
+
+    from cbc_sdk import CBCloudAPI
+    api = CBCloudAPI(url='https://defense.conferdeploy.net', token='ABCDEFGHIJKLMNOPQRSTUVWX/YZ12345678',
+                     org_key='ABCD1234')
+    # as an example, get the list of all policies
```
Comment (avanbrunt-cb) [inline docs/cbc_sdk.cbcloudapi.rst:28]: Would recommend a bit of spacing to make the comment stand out more

File: docs/cbc_sdk.cbcloudapi.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    from cbc_sdk.platform import Policy
```
Comment (avanbrunt-cb) [inline docs/cbc_sdk.cbcloudapi.rst:29]: Given policy is not a common use case for the CBC SDK I would recommend using Alerts as an example

File: docs/cbc_sdk.cbcloudapi.rst (hunk 1 above)
Comment (abowersox-cb) [reply docs/cbc_sdk.cbcloudapi.rst:29]: You got it.

File: docs/cbc_sdk.cbcloudapi.rst (hunk 2 above)
Comment (abowersox-cb) [reply docs/cbc_sdk.cbcloudapi.rst]: I used `WatchlistAlert`, hopefully that's valid here.

**Review by avanbrunt-cb — APPROVED** [review]

**Review by emitreva-cb — APPROVED** [review]

**Review by kebringer-cb — APPROVED** [review]

**Pull request merged**